app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -94,7 +95,8 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
             "klar_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.KLAR.productName),
             "fxa_signed_in" to application.settings().fxaSignedIn,
             "fxa_has_synced_items" to application.settings().fxaHasSyncedItems,
-            "search_widget_installed" to application.settings().searchWidgetInstalled
+            "search_widget_installed" to application.settings().searchWidgetInstalled,
+            "Fenix" to true
```

Comment:
nit: Not sure if it matters but all other keys here seem to be lowercase.

Reply:
This is my fault. I asked for a `Fenix` attribute. If this has not landed yet, then lets make it consistent please.